reana_cluster/backends/kubernetes/k8s.py

```diff
@@ -398,6 +405,39 @@ def init(self):
 
         return True
 
+    def initialize_traefik(self):
+        """Install and initialize traefik via Heml.
+
+        Traefik dashboard service is not accessible by default, to make it
+        accessible inside Minikube service type is changed to NodePort.
+        """
+        from reana_cluster.config import traefik_configuration_file_path
+        try:
+            namespace = 'kube-system'
+            label_selector = 'app=traefik'
+            cmd = ('helm install stable/traefik --namespace {}'
```

Comment:
There is a missing space here.
```suggestion
            cmd = ('helm install stable/traefik --namespace {} '
```